SimeonInstaller.ps1

```diff
@@ -2014,6 +2014,29 @@ CRLFOption=CRLFAlways
                             pipelines = @(@{ authorized = $true; id = $($pipeline.id) })
                         } | ConvertTo-Json -Depth 100) | Out-Null }
 
+                $identities = irm @restProps "https://dev.azure.com/$Organization/_apis/IdentityPicker/Identities" -Method Post -Body @"
+                {
+                    "query": "Contributors",
+                    "identityTypes": [
+                        "group"
+                    ],
+                    "operationScopes": [
+                        "ims",
+                        "source"
+                    ],
+                    "options": {
+                        "MinResults": 1,
+                        "MaxResults": 1000
+                    },
+                    "properties": [
```

Comment:
Just use the loop below, but allow it to return both users and groups in the identityTypes